### ISSUE TITLE
Rename ubuntu-server.js

### ISSUE THREAD

**Issue by vkjr** [status-im/react-native-desktop#238]
Rename `ubuntu-sever.js` because now we use not only ubuntu. `rn-desktop-server.js` for example would be more informative

**Comment by madhavarshney** [comment]
@vkjr - What exactly does this server do?

**Comment by madhavarshney** [comment]
Should I do this in #259?

**Comment by vkjr** [comment]
@madhavarshney, no, please keep every PR as small as possible to reduce review time.
